sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 ## Release History
 
-### 4.34.0-beta.1 (Unreleased)
+### 4.34.0 (2022-08-05)
 
 #### Features Added
 * GA of `DedicatedGatewayRequestOptions` API. See [PR 30142](https://github.com/Azure/azure-sdk-for-java/pull/30142)
 
-#### Breaking Changes
-
-#### Bugs Fixed
-
 #### Other Changes
 * Added `requestSessionToken` to `CosmosDiagnostics` - See [PR 29516](https://github.com/Azure/azure-sdk-for-java/pull/29516)
 * Remove requires `io.netty.transport.epoll` from `module-info`.
```

Comment:
not related to this PR, my bad that forget to add the PR link for the epoll module change: https://github.com/Azure/azure-sdk-for-java/pull/29509